Sources/StripeKit/Connect/Accounts/AccountRoutes.swift

```diff
@@ -294,10 +294,14 @@ public struct StripeConnectAccountRoutes: AccountRoutes {
                        externalAccount: Any?,
                        individual: [String: Any]?,
                        metadata: [String: String]?,
-                       requestedCapabilities: [String],
+                       requestedCapabilities: [String]?,
                        settings: [String: Any]?,
                        tosAcceptance: [String: Any]?) -> EventLoopFuture<StripeConnectAccount> {
-        var body: [String: Any] = ["requested_capabilities": requestedCapabilities]
+        var body = [String: Any]()
```

Comment:
Can we use the explicit type when declaring this variable to follow the style in all the other routes? 
`var body: [String: Any] = [:]`

Reply:
Ok